### ISSUE TITLE
Working example

### ISSUE THREAD

**Issue by sbmadhav** [react-navigation/react-navigation-redux-helpers#78]
The full working example link is not available any more - https://github.com/react-navigation/react-navigation/tree/master/examples/ReduxExample

Is there another example? I am trying to move from react-navigation 1.0.9 beta to 3.3 and it has been a bear so far. :(

**Comment by MayeuldP** [comment]
Same here, working on a project, setting up react-navigation-redux-helpers and all the stuuf (react-navigation,,,,) is a complete mess now

**Comment by Ashoat** [comment]
Good point... we should move that old example to this repo and update it to match the current versions of all the packages.

I unfortunately don't have any spare cycles to do this right now. Any chance either of you could do this and put up a PR? It would be very appreciated!

**Comment by sbmadhav** [comment]
@Ashoat - Do you have the full working example. The above link is not available as pointed out earlier. I can update it.

**Comment by Ashoat** [comment]
https://github.com/react-navigation/react-navigation/tree/84de94d96130b1879fa22dfb29382254cf5d109a/examples/ReduxExample

**Comment by imbudhiraja** [comment]
Need working sample of this -> https://github.com/actionnetworkhq/react-navigation-redux-helpers#back-button

**Comment by imbudhiraja** [comment]
Hey @sbmadhav @MayeuldP ,
I have created a Bolierplate with react-navigation with redux integration. 
Here it is - https://github.com/imbudhiraja/react-native-basekit

**Comment by summerkiflain** [comment]
also looking for working expo app example with the latest versions.

**Comment by bwoodlt** [comment]
@imbudhiraja does that implementation work for you? In your `createStackNavigator`, you were passing `routes` which I believe should be part of the navigator already, not sure why yours implemented this way
